src/xdocs/checks/naming/localvariablename.xml.template

```diff
@@ -13,6 +13,11 @@
           Checks that local, non-<code>final</code> variable names conform to a specified pattern.
           A catch parameter is considered to be a local variable.
         </p>
+        <p>
+          This check does not support pattern variables. Instead, use
+          <a href="https://checkstyle.org/checks/naming/patternvariablename.html#PatternVariableName">
```

Comment:
links in our documentation must be relative if they are for our own domain.

Reply:
The relative link is not working can you correct me ?

Reply:
@rnveach please help the link is not working now. Can you help me what is wrong?

Reply:
Yeah, It should work now. but still link check plugin is falling is this expected? I think it don't recognize the relative paths and always violate them right?

Reply:
> `+<a href="patternvariablename.html#PatternVariableName">patternvariablename.html#PatternVariableName</a>: doesn't exist.`
`+<a href="patternvariablename.html#PatternVariableName">patternvariablename.html#PatternVariableName</a>: doesn't exist.`

These shouldn't be showing up.

Reply:
@mahfouz72 ping. This is still not done.